core/domain/email_manager.py

```diff
@@ -254,6 +257,68 @@ def log_new_error(*args, **kwargs):
     }
 }
 
+NOTIFY_ADMINS_REVIEWERS_NEEDED_EMAIL_DATA = {
```

Comment:
REVIEWER_BOTTLENECK_EMAIL_DATA or similar.

Also, I just thought of the word "shortage", which might be clearer / more specific than "bottleneck" here. Perhaps weigh those two and use the better one throughout this PR?

Reply:
ADMIN_NOTIFICATION_FOR_REVIEWER_SHORTAGE_EMAIL_DATA (the other ones have notification in the name)